fdbserver/StorageMetrics.actor.h

```diff
@@ -209,9 +209,11 @@ struct StorageServerMetrics {
 	// Notifies waiting WaitMetricsRequests through waitMetricsMap, and updates metricsAverageQueue and metricsSampleMap
 	void notify( KeyRef key, StorageMetrics& metrics ) {
 		ASSERT (metrics.bytes == 0); // ShardNotifyMetrics
-		TEST (metrics.bytesPerKSecond != 0); // ShardNotifyMetrics
-		TEST (metrics.iosPerKSecond != 0); // ShardNotifyMetrics
-		TEST(metrics.bytesReadPerKSecond != 0); // ShardNotifyMetrics
+		if (g_network->isSimulated()) {
+			TEST (metrics.bytesPerKSecond != 0); // ShardNotifyMetrics
+			TEST (metrics.iosPerKSecond != 0); // ShardNotifyMetrics
+			TEST(metrics.bytesReadPerKSecond != 0); // ShardNotifyMetrics
```

Comment:
The space is inconsistent. maybe forget to run clang-format? ;)

Reply:
hmm somehow the clang format git commit hook didn't catch this.